### PR TITLE
External Links content

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _docs/style/text-conventions/external-internal-links.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-6-md hxSpan-6-lg" %}
+
+## General guidelines
+
+When you add links from a control page to internal or external content, consider the user's context and information needs when deciding whether to open the contents of a link in the current tab, or in another tab.
```
Comment (palindromebart) [inline _docs/style/text-conventions/external-internal-links.md]: Use Links style for any links in a control panel page. (xref typography)
Links are only to be use for navigation
Be sure you consider the users  context and information needs when deciding proper links behavior for your use case.

File: _docs/style/text-conventions/external-internal-links.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+We recommend that you not open a tab unless it is warranted. User experience is compromised when a new tab opens unnecessarily.
+
+- If a link provides supplemental information that is useful to the user in completing the current workflow, and the user expects to return to the current tab, consider opening the content in another tab. For example, open help documentation in another tab.
+- If the user wants to change contexts, then open the content in the current tab. For example, when the user wants to navigate to a different area of the application, open the link in the current tab.
+
+When a link opens a tab:
+
+- Use the external link icon to visually indicate to the user that a new tab opens.
+- Navigate the user to the new tab. Users should also be able to left-click a link and open a tab without directly navigating to that tab.
+
```
Comment (palindromebart) [inline _docs/style/text-conventions/external-internal-links.md]: When the link will navigate to a new domain, include an external link icon, regardless of whether it opens in a new tab.

File: _docs/style/text-conventions/external-internal-links.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+See [Typography]({{site.baseurl}}/style/typography.html#link) for information about styling links.
```
Comment (palindromebart) [inline _docs/style/text-conventions/external-internal-links.md]: As you'll see in the first comment, I bumped this up to the top.

File: _docs/style/text-conventions/external-internal-links.md (hunk 1 above)
Comment (chri2547) [reply _docs/style/text-conventions/external-internal-links.md]: Done!

File: _docs/style/text-conventions/external-internal-links.md (hunk 2 above)
Comment (chri2547) [reply _docs/style/text-conventions/external-internal-links.md]: Done.

File: _docs/style/text-conventions/external-internal-links.md (hunk 3 above)
Comment (chri2547) [reply _docs/style/text-conventions/external-internal-links.md]: Got it! +1

**Comment by cyrichardson** [comment]
@chri2547 
[Helix_Link-guidelines-cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2980614/Helix_Link-guidelines-cyrEdits.docx)

File: _docs/style/text-conventions/external-internal-links.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-6-md hxSpan-6-lg" %}
+
+## General guidelines
+
+Use the [links style]({{site.baseurl}}/style/typography.html#link) for control panel page links. Use internal links for navigation. Consider the user's context and information needs when you decide how links should behave.
+
+We recommend that the link does not open a tab unless it is warranted. Opening a new tab unnecessarily compromises user experience. The following examples might help you decide when to open a tab:
+
+- If a link provides supplemental information that is useful to the user in completing the current workflow, and the user expects to return to the current tab, consider opening the content in another tab. For example, open help documentation in another tab.
```
Comment (palindromebart) [inline _docs/style/text-conventions/external-internal-links.md]: maybe tweak to "return to the origin tab to continue working"

File: _docs/style/text-conventions/external-internal-links.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- If the user wants to change contexts, then open the content in the current tab. For example, when the user wants to navigate to a different area of the application, open the link in the current tab.
```
Comment (palindromebart) [inline _docs/style/text-conventions/external-internal-links.md]: Maybe Add: remember to use an external link icon if the user will navigate to a different domain even if it is within the same tab.

File: _docs/style/text-conventions/external-internal-links.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-6-md hxSpan-6-lg" %}
+
+## General guidelines
+
+Use the [links style]({{site.baseurl}}/style/typography.html#link) for control panel page links. Use internal links for navigation. Consider the user's context and information needs when you decide how links should behave.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/text-conventions/external-internal-links.md]: Done.

File: _docs/style/text-conventions/external-internal-links.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+We recommend that the link does not open a tab unless it is warranted. Opening a new tab unnecessarily compromises user experience. The following examples might help you decide when to open a tab:
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/text-conventions/external-internal-links.md]: Done.

File: _docs/style/text-conventions/external-internal-links.md (hunk 4 above)
Comment (chri2547) [reply _docs/style/text-conventions/external-internal-links.md]: Done.

File: _docs/style/text-conventions/external-internal-links.md (hunk 5 above)
Comment (chri2547) [reply _docs/style/text-conventions/external-internal-links.md]: Done.

**Comment by palindromebart** [comment]
LGTM!

**Pull request merged**